output.tf

```diff
@@ -3,50 +3,55 @@
 ################################################################################
 # Cluster
 ################################################################################
+output "use_existing_cluster" {
+  description = "Flag to check if you are using an alreday existing cluster"
```

Comment:
description = "Flag to check if you are using an already existing cluster"

Reply:
nice catch